src/objects/culling.ts

```diff
@@ -55,6 +55,11 @@ export async function cull(): Promise<void> {
 }
 
 export async function resumeCulling(): Promise<void> {
+  const application = await waitForApplication()
+
+  if (cullInstance) {
+    cullInstance.cull(application.renderer.screen)
+  }
```

Comment:
This function needs to be removed. There isn't actually a concept of "pause and resume" There's just culled and unculled. So we should rename the `pauseCulling` function `uncull` and then call `uncull` and `cull`.

Reply:
Went ahead and renamed pauseCulling to uncull and updated. Removed resumeCull. 👍